### PR TITLE
Add rough prize/achievements page for EL

### PR DESCRIPTION
**Pull request by hndrewaall**

### CODE REVIEW

File: app/league/dashboard/models.py [hunk 1]
```diff
@@ -324,6 +324,188 @@ def episode_stats(cls, episode=None, season=None, num_players=5):
...
+                         for p in players}
+
+        games = [game for game in cls.query.all()
+                 if game.season == season]
+        for game in games:
+            if game.winner is Color.white:
+                wins[game.white.id] = wins.get(game.white.id, 0) + 1
+                wins_per_ep[game.white.id][game.episode] = \
+                    wins_per_ep[game.white.id][game.episode] + 1
+
```
Comment (wisygig) [inline app/league/dashboard/models.py]: Why +1?

Comment (wisygig) [inline app/league/dashboard/models.py]: As opposed to += 1?

Comment (hndrewaall) [inline app/league/dashboard/models.py]: oh I don't know

Comment (hndrewaall) [inline app/league/dashboard/models.py]: I'll fix those

**Comment by hndrewaall** [comment]
Force-pushed post-rebase

**Pull request merged**